### DISCUSSION TITLE
Web Support

### DISCUSSION THREAD

**Discussion by caleb654** [TNorbury/command_palette#10] [Ideas]
I tested the example project on web and it seems to work great. It just throws some exceptions but nothing breaks. Why doesn't the package show that it supports web? Have you found issues with it?

**Comment by TNorbury** [comment]
I think that's determined automatically (by default). This is the first package of mine that's never been given web automatically. I'll update the pubspec to explicitly support all platforms 👍🏻

**Comment by TNorbury** [comment]
I've pushed the change the pub.dev
